# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../log4cxx/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- log4cxx-0.9.7/src/socketoutputstream.cpp.vanilla	2009-10-22 09:45:32.000000000 +0200
+++ log4cxx-0.9.7/src/socketoutputstream.cpp	2009-10-22 09:45:54.000000000 +0200
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+#include <cstring>
  
 #include <log4cxx/helpers/socketoutputstream.h>
 #include <log4cxx/helpers/socket.h>
--- log4cxx-0.9.7/src/stringtokenizer.cpp.vanilla	2009-10-22 10:03:12.000000000 +0200
+++ log4cxx-0.9.7/src/stringtokenizer.cpp	2009-10-22 10:03:45.000000000 +0200
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+#include <cstring>
  
 #include <log4cxx/helpers/stringtokenizer.h>
 
--- log4cxx-0.9.7/src/socketinputstream.cpp.vanilla	2009-10-22 10:04:37.000000000 +0200
+++ log4cxx-0.9.7/src/socketinputstream.cpp	2009-10-22 10:04:46.000000000 +0200
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+#include <cstring>
  
 #include <log4cxx/helpers/socketinputstream.h>
 #include <log4cxx/helpers/socket.h>
--- log4cxx-0.9.7/src/xmlsocketappender.cpp.vanilla	2009-10-22 10:12:43.000000000 +0200
+++ log4cxx-0.9.7/src/xmlsocketappender.cpp	2009-10-22 10:12:50.000000000 +0200
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include <cstring>
+
 #include <log4cxx/net/xmlsocketappender.h>
 #include <log4cxx/helpers/loglog.h>
 #include <log4cxx/helpers/socketoutputstream.h>
--- log4cxx-0.9.7/src/timezone.cpp.vanilla	2009-10-22 10:12:12.000000000 +0200
+++ log4cxx-0.9.7/src/timezone.cpp	2009-10-22 10:12:19.000000000 +0200
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+#include <cstring>
  
 #include <log4cxx/helpers/timezone.h>
 #include <locale>
--- log4cxx-0.9.7/tests/src/drfatestcase.cpp.vanilla	2009-10-22 10:13:07.000000000 +0200
+++ log4cxx-0.9.7/tests/src/drfatestcase.cpp	2009-10-22 10:13:13.000000000 +0200
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+#include <cstring>
+
 #include <cppunit/TestFixture.h>
 #include <cppunit/extensions/HelperMacros.h>

